tests/test_usg.py

```diff
@@ -1,13 +1,15 @@
 import datetime
+import configparser
 
 import pytest
 from pytest import MonkeyPatch
 
+import usg
```

Comment:
perhaps we do here like the other tests where we import what is needed and not use the `usg.<>`  call?
Just to follow the same pattern across the code.